Handlers/ShowroomHandlers.js

```diff
@@ -894,6 +894,128 @@ function updateScheduleEvent (req, res, next) {
+            //     if (error) {
+            //         logError(error, logCtx);
+            //         errorStatus = 400;
+            //         errorMsg = error.message;
+            //     }
+                callback(null);
+            // });
+        },
+        function (callback) {
+            //Persist updated event to DB
+            console.log("time: "+req.params.time);
+            let time = new Date(parseInt(req.params.time));
+            var upcoming = true;
+            var all = false;
+            var date = req.body.e_date;
+
+            showroomDB.getEvents(all, false, upcoming, time, date, (error, result) => {
+                if (error) {
+                    errorStatus = 500;
+                    errorMsg = error.toString();
+                    logError(error, logCtx);
+                    callback(error, null);
+                } else if (result == undefined || result == null) {
+                    errorStatus = 404;
+                    errorMsg = "No events found.";
+                    logError(error, logCtx);
+                    callback(new Error(errorMsg), null);
+                } else {
+                    log("Response data: " + JSON.stringify(result), logCtx);
+                    callback(null, result);
+                }
+            });
+        },
+        function(result, callback){
+            var event = req.body;
+            let time = new Date(result[0].starttime);
+            let duration = new Date(result[0].duration);
+            var delta = +new Date(event.starttime) - +time + (+new Date(event.duration) - +duration) * 60000;
+            var updatedEventList = [];
+
+            updatedEventList.push(event);
+
+            
+
+            for(let i = 1; i < result.length; i++){
+                event = result[i];
+
+                // let updatedEvent = {
+                //     meetid: event.meetid,
+                //     adminid: event.adminid,
+                //     starttime: event.starttime + delta,
+                //     duration: event.duration,
+                //     title: event.title,
+                //     projectid: event.projectid,
+                //     e_date: event.e_date,
+                // };
+                console.log("event: ");
+                console.log(event);
+                console.log("event starttime: ");
+                console.log(new Date(event.starttime));
+                console.log("delta: ");
+                console.log(delta);
+
+                let updatedEvent = {
+                    ...event,
+                    starttime : new Date(+new Date(event.starttime) + delta)
+                };
+
+                updatedEvent.starttime = new Date(+new Date(event.starttime) + delta)
+
+                console.log("updatedEvent: ");
+                console.log(updatedEvent);
+
+                updatedEventList.push(updatedEvent);
+                
+            }
+
+            var event = req.body; //JSON object of event to be updated
```

Comment:
esto hace lo mismo que la linea 940?

Reply:
The var event is unused after line 983, removed var reassignment